docs: update volume add warning about delete rotate key (#2738)
bareos/bareos: docs/manuals/source/TasksAndConcepts/BareosConsole.rst

   volume=<volume-name> encrypt=<yes/no/rotate>` for :ref:`scsicrypto-sd`.

   .. warning::
      - As a new encryption key will only be loaded on volume mount, make sure you
        re-mount any currently mounted volume when changing the encryption key!

>       - Removing or rotating a key need purging the volume first.

Reviewer: ```suggestion
      - Removing or rotating a key requires the volume to be purged.
```

Author: done
